ucey-star/ID8
# console log cleanup

## Description
Replaced all `console.log/error` statements with user-facing `Snackbar` notifications across multiple components to improve user experience. The changes include:

- Updated existing Snackbar states to handle info/warning messages
- Added user-friendly notifications for key events (project fetch, updates, errors)
- Maintained all existing functionality while improving error visibility
- Standardized error handling with descriptive messages

**No changes were made to the core business logic or component structure. This update purely focuses on improving error visibility and user feedback.**

## Type of Change


## Testing Instructions
To test this ensure that all app functionalities work as expected first to ensure no part of the application is breaking. Secondly on pages where data is being fetched one can input the wrong `userId` or `ProjectId` to trigger the error cases. This is not entirely necessary as I will personally fix any bug introduced by this chang. 

## Screenshots (if applicable)

If your changes include visual components, add screenshots showing the affected pages or elements.
